mcux/drivers/imx/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ zephyr_compile_definitions_ifdef(
 zephyr_library_compile_definitions_ifdef(
   CONFIG_HAS_MCUX_CACHE FSL_SDK_ENABLE_DRIVER_CACHE_CONTROL
 )
+zephyr_library_sources(fsl_common.c)
 
```

Comment:
Why is this needed?

Reply:
In fact, fsl_common.c/h provide developer  many common functions to use. In our sdk, all rt series project will include this file.  On RT1170 board, this file must be  compiled into image because PMU unit need delay function to ensure power of PLL unit stable.